controller/apps.go

```diff
@@ -61,36 +59,43 @@ func getAndCheckOSIOClientV1(ctx context.Context) *OSIOClientV1 {
 	if os.Getenv("FABRIC8_WIT_API_URL") != "" {
```

Comment:
Why is this not coming from Configuration?

Reply:
This was discussed with Xavier at the time.  (December)
It is actually more of a testing hook than a production hook at this point.